bazel/foreign_cc/BUILD

```diff
@@ -11,7 +11,10 @@ configure_make(
         "--enable-shared=no",
         "--enable-frame-pointers",
         "--disable-libunwind",
-    ],
+    ] + select({
```

Comment:
@keith I believe this is workaround for https://github.com/bazelbuild/bazel/pull/9154

Reply:
We should update rules_foriegn_cc instead where this is fixed

Reply:
We should fix it there too then 😞